code/game/objects/items/weapons/shields.dm

```diff
@@ -58,7 +58,7 @@
 
 /obj/item/shield/energy
 	name = "energy combat shield"
-	desc = "A shield capable of stopping most melee attacks. Protects user from almost all energy projectiles. It can be retracted, expanded, and stored anywhere."
+	desc = "A handheld shield projector which reflects energy bolts but will not stop other attacks. It can be retracted, expanded, and stored anywhere."

```

Comment:
`desc = "A shield that reflects almost all energy projectiles, but is useless against physical attacks. It can be retracted, expanded, and stored anywhere."`